extensions/data-workspace/src/dataworkspace.d.ts

```diff
@@ -33,6 +33,11 @@ declare module 'dataworkspace' {
 		 */
 		showProjectsView(): void;
 
+		/**
+		 * Refreshes the projects tree
+		 */
+		refreshProjectsTree(): void;
```

Comment:
I'd suggest making the comment clearer that this only fires the event to refresh the tree, the tree is not guaranteed to be refreshed after the call returns.